### PR TITLE
[netatmo] Handle new event types

### PR DESCRIPTION
**Pull request by clinique**
I identified two events sent through the webhook related to : 
* Manual action on the thermostat
* Alarm triggered on a weather station device

These new events implied a change of type for the subType identifier because they hold string values.

Discovering these new webhook events (that was restricted to Security) makes me think that webhook handling will need a complete revamp in the future for better handling of various cases but at least now, they are recognized.

### CODE REVIEW

**Review by lolodomo — CHANGES_REQUESTED** [review]
Only a minor comment

**Comment by lolodomo** [comment]
I am very interested by the new RAIN alarm. Does it require to have webhook setup or will it be produced when events are fetched even without webhook connected ?

**Comment by clinique** [comment]
@lolodomo : it is only going through webhooks.

File: bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/EventType.java [hunk 1]
```diff
@@ -143,7 +143,25 @@ public enum EventType {
     NEW_DEVICE(ModuleType.HOME),
 
     @SerializedName("co_detected")
-    CO_DETECTED(ModuleType.CO_DETECTOR);
+    CO_DETECTED(ModuleType.CO_DETECTOR),
+
+    @SerializedName("alarm_event") // an alarm alarm event arrived on a weather station module
```
[the remark this answers is not in the document]
Comment (clinique) [reply bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/EventType.java]: Done

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Comment by lolodomo** [comment]
> @lolodomo : it is only going through webhooks. I guess it may be the same mechanism used by the mobile app to generate notification of the ongoing weather events.

It would have been cool if such event could have been caught even without setting up webhook.

**Comment by clinique** [comment]
Yes, it took me some times to setup something properly, enough resiliant to avoid being banned.

**Pull request merged**